firestore-palm-summarize-text/functions/src/config.ts

```diff
@@ -15,20 +15,31 @@
  */
 
 export interface Config {
+  location: string;
+  projectId: string;
+  instanceId: string;
   collectionName: string;
   textField: string;
   responseField: string;
   targetSummaryLength?: number;
+  useVertex?: boolean;
```

Comment:
Wdyt this naming updated to something like `provider`.

If there were to many any future model updates, then this would have to be renamed?

Reply:
This would have to be renamed if there was some other provider (aside from GL or Vertex) which would provide the PaLM API - in that case we'd have to handle the SDK of that new provider. I can change the name of the variable

Reply:
From the users perspective they won't see "useVertex", they will see a select box with both options